### DISCUSSION TITLE
Welcome to 100LinesOfCode Discussions!

### DISCUSSION THREAD

**Discussion by josharsh** [josharsh/100LinesOfCode#385] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by TasleemaShaik** [comment]
Hello there,
I just wanted to check is there anyone(maintainers) actively looking into pull requests?
I'm bit lost because I can see there are lot of pull requests from long back, is this because these are not good ones or someone has to approve and merge them?
Thanks,
Tasleema.

**Comment by aliasgarsogiawala** [comment]
Hey @TasleemaShaik not to worry , Im actively looking into PRs and merging them

**Reply by TasleemaShaik** [reply]
Awesome, Thank you so much @aliasgarsogiawala .... for helping merging my PR.

Thanks,
Tasleema.
